Hardware selection guide (#656)
jellyfin/jellyfin.org: docs/general/administration/hardware-selection.md

---
uid: admin-hardware-selection
title: Hardware Selection
---

# Selecting Appropriate Hardware

> This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (ex. Hardware Acceleration).

Reviewer: ```suggestion
This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (e. g.. hardware acceleration).
```

## TL;DR

For a Jellyfin server, we recommend the following:

### Normal Server From Scratch

> - CPU: 4 Threads

Reviewer: ```suggestion
- CPU: 4 threads
```

>   - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.

Reviewer: Where is this coming from?

Author: 4 threads is what I think should be able to comfortably handle most software decoding tasks in legacy formats.
Atom, Celeron/Pentium N/J series was excluded because they are low powered parts that may struggle in some tasks but correct me if I'm wrong here.

Author: This line no longer exists in PR and I will close this

>   - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.

Reviewer: ```suggestion
  - Please Avoid 11th gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
```

>   - Please Avoid Rapsberry PIs and other low powered SBCs.

Reviewer: ```suggestion
  - Please avoid Rapsberry PIs and other low powered SBCs.
```

>   - Please Avoid Intel Core M / Y series CPUs.

Reviewer: ```suggestion
  - Please avoid Intel Core M / Y series CPUs.
```

- RAM: 8GB or more
- Storage: SSD for Jellyfin files and transcoding cache.
- Graphics: Intel > Nvidia > AMD > Apple.
  - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.

>   - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.

Reviewer: Might be good to add an infobox about the reasoning behind this.

Reviewer 2: I also think this could be phrased a little bit better. This is unclear, and I think the language should be made less direct.

"You should consider Intel ARC over Nvidia" vs "We recommend Intel ARC over Nvidia"

This language would be better throughout.

Author: Addressed in a commit

>   - AMD Graphics or Apple Silicon is NOT recommended due to poor encoder quality.

Reviewer: Is not true for AV1 on AMD anymore, just an FYI

Author: I'll leave a note

  - When Intel based Mac hardware is used, we recommend installing Windows or Linux for the best experience.

### Low Power Applications

For low power applications, we recommend Intel 12th gen or newer Atom CPUs with integrated graphics.

### Upgrading Existing System for use as a Server

We recommend Intel ARC GPUs for upgrading existing systems. If Intel ARC GPUs aren't available, we recommend Nvidia GTX 16 / RTX 20 or newer, excluding the GTX 1650.

## Detailed Guide

### CPU

> The CPU will be responsible for many tasks, Notably:

Reviewer: ```suggestion
The CPU will be responsible for many tasks, notably:
```

- Transcoding Audio

> - Decoding video in unsupported codecs.

Reviewer: ```suggestion
- Decoding video in unsupported codecs
```

> Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.

Reviewer: I don't think this is true if source is 4k HDR

Author: That's fair, I will add some clarification

#### Integrated Graphics

> Integrate graphics can be useful for transcoding video. Please refer to the GPU section for a guide.

Reviewer 2: ```suggestion
Integrated graphics can be useful for transcoding video. Please refer to the GPU section for a guide.
```

#### Resizable BAR

Resizable Bar (ReBAR) is a PCI Express feature that requires support from the platform. This feature is supported on Intel 10th gen or above and AMD Ryzen 3000 Series or above.

When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.

### System Memory (RAM)

> As a general rule of thumb, we recommend at least 8GB for Linux and 16GB for Windows, though significantly smaller amounts may work just fine.

Reviewer: Why double it for Windows?

Author: Windows itself as an OS is more bloated so I don't think it's a bad idea to have more.
It's true that windows with 8GB will work perfectly for a JF server but it can get tight.

### Graphics Cards (GPUs)

> With modern systems, the media engines are usually located on the GPUs. Therefore, the hardware acceleration options available is entirely determined by the choice of GPUs.

Reviewer: ```suggestion
With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.
```

We recommend Intel graphics over Nvidia graphics because of the quality. We recommend avoiding AMD or Apple Silicon since they produce poor quality output. Encoder Quality: Intel > Nvidia > AMD > Apple

> The following is a list of commonly used codecs:

Reviewer 2: I honestly think this entire section is unnecessary. Or at least poorly placed. It very well be nice to have in the documentation somewhere, but I'm not sure that it belongs in a hardware guide.

Reviewer: We already have it in the documentation somewhere.

Author: Are you referring to the client codec support page?

Reviewer: Correct

Author: I have changed that table to this link.

Author: While I do agree that the codec section would be better off somewhere else, I do think that having a minimal section here until that codec page exists would be very helpful

- H.264 (AVC): Blu-Ray, most streaming services
- H.265 (HEVC): Blu-Ray UHD
- MPEG-2: DVD
- VC-1: Blu-Ray
- VP8, VP9, AV1: streaming services

> The following is a list of codecs Jellyfin supports transocding to:

Reviewer: In theory not correct, we could transcode to others too. Just a note though.

- H.264 (Most common transcode target)
- H.265 (Limited supported by clients)

For decoding support, the more codecs the better. However, there are only a few codecs that media is commonly available in, as listed above.

#### Intel Graphics

> Please refer to [This Table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.

Reviewer: ```suggestion
Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
```

Do Note that Intel ARC requres kernel version 6.2 or newer on Linux. Please check your distribution for details.

#### Nvidia Graphics

> Please refer to [This Table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.

Reviewer: ```suggestion
Please refer to [this table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.
```

#### AMD Graphics

> Please refer to [This Article](https://en.wikipedia.org/wiki/Video_Coding_Engine) for supported codecs.

Reviewer: ```suggestion
Please refer to [this article](https://en.wikipedia.org/wiki/Video_Coding_Engine) for supported codecs.
```

#### Apple Silicon

> Supported codecs are listed below:

Reviewer: Might want to add 10bit HEVC and VP9 capabilities

Author: This info isn't published by Apple and the info here was pieced together with info from the internet
There are people who tested VP9 to be supported but I don't think anyone tested 10bit VP9.
I have an M1 Pro device on hand so I can test it out on my system for this particular chip if there are test clips available.

Author: I've done some testing and VP9 10bit does hardware decode on my M1 Pro Macbook

| Codec      | M1 | M1 Pro | M2 | M2 Pro |
|------------|----|--------|----|--------|
| H.264      | ✅  | ✅      | ✅  | ✅      |
| H.265      | ✅  | ✅      | ✅  | ✅      |
| ProRes     | ✅  | ✅      | ✅  | ✅      |
| ProRes RAW | ❌  | 🔶      | 🔶  | 🔶      |
| VP9        | ❌  | ❌      | 🔶  | 🔶      |
| AV1        | ❌  | ❌      | ❌  | ❌      |

Refer to the Pro column for Max and Ultra SOCs.

✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.

Please note that many hardware acceleration features are not available on MacOS for Jellyfin, and no Apple Silicon media engine drivers exist for other operating systems.

### Storage

> There are 2 general types of storage that will be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.

Reviewer: ```suggestion
There are two general types of storage that will be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.
```

#### Media Storage

Anything with higher sequential access speed than bitrate of media will work.

#### Jellyfin Files

> Jellyfin files will see lots of random access, therefore we recommend SSDs for Jellyfin files. AVOID mechanical drives that use SMR (Shingled Magnetic Recording) as they will result in a VERY poor experience. (But if you want your JF home page take 10 mins to load then sure.)

Reviewer: ```suggestion
Jellyfin files will see lots of random access, therefore we recommend SSDs for Jellyfin files. AVOID mechanical drives that use SMR (Shingled Magnetic Recording) as they will result in a VERY poor experience.
```

Kinda unnecessary snarky comment even though it it true lol

Reviewer 2: Agreed. Official documentation pages need to keep an air of impartiality and professionalism. The project is represented by these pages.

## TL;DR

For a Jellyfin server, the following is recommended:

### Normal Server From Scratch

- CPU: 4 threads
  - Please Avoid 11th gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
  - Please avoid Rapsberry PIs and other low powered SBCs.
  - Please avoid Intel Core M / Y series CPUs.
- RAM: 8GB or more
- Storage: SSD for Jellyfin files and transcoding cache.
- Graphics: Intel > Nvidia > AMD > Apple.
  - A system with Intel 7th gen or newer Core series CPU with integrated graphics is recommended.
  - When using dedicated graphics, please consider using Intel ARC over Nvidia graphics.
  - When using Nvidia graphics, GTX 16 / RTX 20 (Turing) or newer is recommended, excluding GTX 1650.
  - AMD Graphics or Apple Silicon is NOT recommended due to poor H.264 and H.265(HEVC) encoder quality.

:::note A Note on Apple hardware

It is recommended that Intel-based Macs be used with Windows or Linux installed to host Jellyfin. Many hardware acceleration features aren't available on MacOS due to the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork not being available.

:::

### Low Power Applications

> For low power applications, Intel 12th gen or newer Atom CPUs with integrated graphics are recommended.

Reviewer 3: Link to the LowPower settings would be nice

Author: Where is this located in the docs?

Reviewer 3: https://jellyfin.org/docs/general/administration/hardware-acceleration/intel#low-power-encoding

Author: Linked

#### Resizable BAR

Resizable Bar (ReBAR) is a PCI Express feature that requires support from the platform. This feature is supported on Intel 10th gen or above and AMD Ryzen 3000 Series or above.

When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.

### System Memory (RAM)

As a general rule of thumb, at least 8GB is recommended, though significantly smaller amounts may work just fine. If integrated graphics is being used, please consider adding more as integrated graphics usually use part of the system memory as video memory.

### Graphics Cards (GPUs)

With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.

Encoder Quality: Intel > Nvidia > AMD > Apple

Intel graphics is recommended over Nvidia graphics because of the quality. AMD or Apple Silicon are not recommended since they produce poor quality H.264 and H.265(HEVC) output.

:::note A Note on AMD AV1 Encoders

> With AV1, AMD has significantly improved the quality of their encoders. However, transcoding to AV1 is currently not supported on Jellyfin, and transcoding to H.264 or H.265(HEVC) is still more likely due to AV1 being in the early stages of adoption.

Reviewer 3: To give present accurate description i would rephrase to:
`[...]With jellyfin server version 10.8.10, AV1 codec is not supported[...]`
here instead of below

:::

A list of common codecs can be found [here](/docs/general/clients/codec-support/)

The following is a list of codecs Jellyfin supports transocding to:

- H.264 (Most common transcode target)
- H.265 (Limited supported by clients)

:::note AV1 Support

> Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1 as a target. Source content in AV1 will transcode to other codecs normally if needed. However, this will change in the near future.

Reviewer 3: `might change in the future`

---
uid: admin-hardware-selection
title: Hardware Selection
---

# Selecting Appropriate Hardware

This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (e. g.. hardware acceleration).

## TL;DR

For a Jellyfin server, the following is recommended:

### Normal Server

- CPU: Intel Core i3-2100, AMD FX-4100 or better
- RAM: 8GB or more
- Storage: 32GB SSD storage for Jellyfin files and transcoding cache.
- Graphics: Intel HD 6xx (7th gen integrated graphics) or newer, Nvidia GTX 16 / RTX 20 series or newer (excluding GTX 1650). Intel recommended over Nvidia, AMD and Apple Silicon not recommended.

:::tip Upgrading an Existing System

> Intel ARC GPUs are recommended when upgrading an existing system to be used as a Jellyfin server. If cases where Intel ARC is not feasible, Nvidia graphics is recommended. Please select an appropriate GPU according to the recommended specs above.

Reviewer: ```suggestion
Intel ARC GPUs are recommended when upgrading an existing system to be used as a Jellyfin server. In cases where Intel ARC is not feasible, Nvidia graphics is recommended. Please select an appropriate GPU according to the recommended specs above.
```

:::

:::tip Fully Utilizing Intel-based Macs

It is recommended that Intel-based macs be used with Windows or Linux installed to host Jellyfin. Many hardware acceleration features aren't available on MacOS due to the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork not being available.

:::

### Low Power Applications

For low power applications, Intel 12th gen or newer Atom CPUs with integrated graphics are recommended. It is also recommended that [Low Power Encoding](/docs/general/administration/hardware-acceleration/intel/#low-power-encoding) be setup.

## Detailed Guide

### CPU

The CPU will be responsible for many tasks, notably:

- Transcoding Audio
- Decoding video in unsupported codecs

Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.

:::caution

> Newer codecs can also require software decoding if there are no available hardware decoders. High bitrate HEVC content such as commonly found in 4k HDR can be very demanding even on modern CPUs. Please considering adding a modern GPU with 10bit HEVC decoding support.

Reviewer: Tone-mapping is also a problem (without hwa this is currently not possible)

:::

#### Integrated Graphics

> Integrated graphics can be useful for transcoding video. Please refer to the GPU section for a guide.

Reviewer: Add a link to the section

#### Resizable BAR

> Resizable Bar (ReBAR) is a PCI Express feature that requires support from the platform. This feature is supported on Intel 10th gen or above and AMD Ryzen 3000 Series or above.

Reviewer: Maybe a short explanation what it is and why it helps is useful

When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.

### System Memory (RAM)

> As a general rule of thumb, at least 8GB is recommended, though significantly smaller amounts may work just fine. If integrated graphics is being used, please consider adding more as integrated graphics usually use part of the system memory as video memory.

Reviewer: Note on dual channel memory might be useful here (see hwa guide)

### Graphics Cards (GPUs)

With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.

Encoder Quality: Intel > Nvidia > AMD > Apple

Intel graphics is recommended over Nvidia graphics because of the quality. AMD or Apple Silicon are not recommended since they produce poor quality H.264 and H.265(HEVC) output.

:::note A Note on AMD AV1 Encoders

With AV1, AMD has significantly improved the quality of their encoders. However, Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1. Even after Jellyfin adds support for transcoding to AV1, transcoding to H.264 or H.265(HEVC) is still more likely due to AV1 being in the early stages of adoption.

:::

A list of common codecs can be found [here](/docs/general/clients/codec-support/)

The following is a list of codecs Jellyfin supports transocding to:

- H.264 (Most common transcode target)
- H.265 (Limited supported by clients)

:::note AV1 Support

> Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1 as a target. Source content in AV1 will transcode to other codecs normally if needed. However, this might change in the future.

Reviewer: The last sentences seems to reference the sentence before it but should the first sentence. The decoding capabilities will not change

:::

For decoding support, the more codecs the better. However, there are only a few codecs that media is commonly available in, as listed above.

#### Intel Graphics

Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.

:::caution F series CPUs

> Intel CPUs with a model that ends in F don't have integrated graphics. These are intended to be used with a dedicated graphics card.

Reviewer: ```suggestion
Intel CPUs with a model description that ends with F don't have integrated graphics. These are intended to be used with a dedicated graphics card.
```

:::

:::note Intel 12th Gen, 13th Gen and ARC Support

If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that `jellyfin-ffmpeg6` be installed instead of the default `jellyfin-ffmpeg5` on these hardware platforms.

:::

#### Nvidia Graphics

Please refer to [this table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.

:::caution Low End Cards

> Certain low end cards (e.g. GT 1030) don't have encoding hardware. Please be careful when choosing a GPU.

Reviewer: Is the proper name really `GT 1030`?

:::

#### AMD Graphics

Please refer to [this article](https://en.wikipedia.org/wiki/Video_Coding_Engine) for supported codecs.

:::caution Low End Cards

Certain low end cards (e.g. RX 6400, RX 6500) don't have encoding hardware. Please be careful when choosing a GPU.

:::

:::caution Integrated Graphics

> Many AMD CPUs don't have integrated graphics. These are intended to be used with a dedicated graphics card. Please check the product page for each CPU for more information.

Reviewer: Might want to add a note which CPU series include a GPU (e. g. the `G` denoted and Zen4)

:::

#### Apple Silicon

Supported codecs are listed below:

| Codec       | M1 | M1 Pro | M2 | M2 Pro |
|-------------|----|--------|----|--------|
| H.264 8bit  | ✅  | ✅      | ✅  | ✅      |
| H.264 10bit | 🔶  | 🔶      | 🔶  | 🔶      |
| H.265 8bit  | ✅  | ✅      | ✅  | ✅      |
| H.265 10bit | ✅  | ✅      | ✅  | ✅      |
| VP9 8bit    | 🔶  | 🔶      | 🔶  | 🔶      |
| VP9 10bit   | 🔶  | 🔶      | 🔶  | 🔶      |
| AV1         | ❌  | ❌      | ❌  | ❌      |

Refer to the Pro column for Max and Ultra SOCs.

✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.

> 

Reviewer: We might just want to move this to the codecs page and reference it here

Author: I think this can stay here for now as the codec page is intended for client side. I do think that it won't be a bad idea to have a makeover of the codecs page

:::caution

Many hardware acceleration features are not available on macOS for Jellyfin, as the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork isn't available for macOS. No Apple Silicon media engine drivers exist for other operating systems currently. You will NOT be able to use hardware acceleration if you are running [Asahi Linux](https://asahilinux.org/).

:::

### Storage

> There are two general types of storage that will be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.

Reviewer: ```suggestion
There are two general types of storage that can be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.
```

Reviewer: Or `should` as an alternative

---
uid: admin-hardware-selection
title: Hardware Selection
---

# Selecting Appropriate Hardware

This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (e. g. hardware acceleration).

## TL;DR

For a Jellyfin server, the following is recommended:

### Normal Server

- CPU (With dGPU): Intel Core i5-2300, AMD Ryzen 3 1200 or better. (Geekbench 6 Multicore 1500 or better)
- CPU (Without dGPU): Intel Pentium G4560, Intel Core i3-7100 or better. (Intel 7th gen or newer Pentium or better)
- RAM: 8GB or more

> - Storage: 32GB SSD storage for Jellyfin files and transcoding cache.

Reviewer: We might want to bump that. 32GB for cache and file might be a bit on the lower side (e.g. trancoding 4k might already eat 20GB)

Author: I bumped it to 60GB.

:::caution Low End Cards

Certain low end cards (e.g. RX 6400, RX 6500) don't have encoding hardware. Please be careful when choosing a GPU.

:::

:::caution Integrated Graphics

Many AMD CPUs don't have integrated graphics. These are intended to be used with a dedicated graphics card.

For Ryzen 1000 ~ 5000 Series, desktop CPUs with a model description that ends with G have integrated graphics while other desktop CPUs do NOT. For Ryzen 7000 Series, desktop CPUs with a model description that ends with F do NOT have integrated graphics while others do.

Please check the product page of your CPU for more info.

:::

#### Apple Silicon

Supported codecs are listed below:

| Codec       | M1 | M1 Pro | M2 | M2 Pro |
|-------------|----|--------|----|--------|
| H.264 8bit  | ✅  | ✅      | ✅  | ✅      |
| H.264 10bit | 🔶  | 🔶      | 🔶  | 🔶      |
| H.265 8bit  | ✅  | ✅      | ✅  | ✅      |
| H.265 10bit | ✅  | ✅      | ✅  | ✅      |
| VP9 8bit    | 🔶  | 🔶      | 🔶  | 🔶      |
| VP9 10bit   | 🔶  | 🔶      | 🔶  | 🔶      |
| AV1         | ❌  | ❌      | ❌  | ❌      |

Refer to the Pro column for Max and Ultra SOCs.

> ✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.

Reviewer: See other comments, might be better suited for an advanced section.

Author: I think this can stay here for now for the lack of a better place to put it without pouring hours into writing a dedicated apple silicon guide which realistically isn't recommend for hosting JF anyway.

### Normal Server From Scratch

- CPU: 4 Threads
  - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
  - Please Avoid Rapsberry PIs and other low powered SBCs.
  - Please Avoid Intel Core M / Y series CPUs.
- RAM: 8GB or more
- Storage: SSD for Jellyfin files and transcoding cache.
- Graphics: Intel > Nvidia > AMD > Apple.
  - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.
  - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.
  - AMD Graphics or Apple Silicon is NOT recommended due to poor encoder quality.

>   - When Intel based Mac hardware is used, we recommend installing Windows or Linux for the best experience.

Reviewer 2: As above: "is recommended" over "we recommend" here is better. It might also be worth thinking of framing it in terms of WHY it might be recommended, instead of just saying that X or Y is recommended. For example: "When using Intel based Mac hardware, installing Windows or Linux enables features that Mac OS does not, and will better support transcoding and hardware acceleration."

Author: This section is meant to be a TL;DR so I think the explanation should live somewhere else.

Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.

:::note Intel 12th Gen, 13th Gen and ARC Support

> If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that jellyfin-ffmpeg6 be installed instead of the default jellyfin-ffmpeg5 on these hardware platforms.

Reviewer: ```suggestion
If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that `jellyfin-ffmpeg6` be installed instead of the default `jellyfin-ffmpeg5` on these hardware platforms.
```

✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.

:::caution

> Many hardware acceleration features are not available on MacOS for Jellyfin, as the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork isn't available for MacOS. No Apple Silicon media engine drivers exist for other operating systems currently. You will NOT be able to use hardware acceleration if you are running [Asahi Linux](https://asahilinux.org/).

Reviewer: ```suggestion
Many hardware acceleration features are not available on macOS for Jellyfin, as the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork isn't available for macOS. No Apple Silicon media engine drivers exist for other operating systems currently. You will NOT be able to use hardware acceleration if you are running [Asahi Linux](https://asahilinux.org/).
```
